packages/runtime/src/components/builtin/Text/register.ts

```diff
@@ -36,11 +37,12 @@ export function registerComponent(runtime: ReactRuntime) {
                               id: null,
                               style: [
                                 {
-                                  deviceId: 'mobile',
+                                  // TODO: This will only work if they have the mobile breakpoint.
```

Comment:
Can we make this more robust by only adding the entry to the array if the breakpoints contain the desired hardcoded ID?

Reply:
Updated